modules/installation-aws-about-government-region.adoc

```diff
@@ -2,22 +2,47 @@
 //
 // * installing/installing_aws/installing-aws-government-region.adoc
 
-[id="installation-aws-about-government-region_{context}"]
-= AWS government and secret regions
+ifeval::["{context}" == "installing-aws-government-region"]
+:aws-gov:
+endif::[]
+ifeval::["{context}" == "installing-aws-secret-region"]
+:aws-secret:
+endif::[]
+
+[id="installation-aws-about-gov-secret-region_{context}"]
+ifdef::aws-gov[]
+= AWS government regions
+endif::aws-gov[]
+ifdef::aws-secret[]
+= AWS secret region
+endif::aws-secret[]
 
 {product-title} supports deploying a cluster to
 link:https://aws.amazon.com/govcloud-us[AWS GovCloud (US)] regions and the link:https://aws.amazon.com/federal/us-intelligence-community/[AWS Commercial Cloud Services (C2S) Secret Region]. These regions are specifically designed for US government agencies at the federal, state, and
 local level, as well as contractors, educational institutions, and other US
 customers that must run sensitive workloads in the cloud.
 
-These regions do not have published {op-system-first} Amazon Machine Images (AMI) to select, so you
+ifdef::aws-secret[]
+The C2S Secret Region does not have a published {op-system-first} Amazon Machine Images (AMI) to select, so you
 must upload a custom AMI that belongs to that region.
+endif::aws-secret[]
 
+ifdef::aws-gov[]
 The following AWS GovCloud partitions are supported:
 
 * `us-gov-west-1`
 * `us-gov-east-1`
```

Comment:
This is well outside the scope of this PR, but the order of these are interesting.
```suggestion
* `us-gov-east-1`
* `us-gov-west-1`
```

Reply:
Completely unintentional. Looks like when I was separating content between gov and secret, I reordered them.

Reply:
> Did you want to reorder and have east come before west? Right now, west comes before east.

Yes. Thank you.